cdap-ui/app/cdap/styles/common.scss

```diff
@@ -192,6 +192,32 @@ body {
     }
   }
 
+  .grid.grid-container.grid-compact {
+    .grid-header,
+    .grid-body {
+      .grid-row > div {
+        padding: 5px 7px;
+        min-height: 29px;
```

Comment:
nit: Do we need this min-height?

Reply:
Yes we do, otherwise it defaults to 34px